jellyfin-model/src/commonMain/kotlin/org/jellyfin/sdk/model/extensions/GeneralCommandMessageExtensions.kt

```diff
@@ -0,0 +1,18 @@
+package org.jellyfin.sdk.model.extensions
+
+import org.jellyfin.sdk.model.socket.GeneralCommandMessage
+import kotlin.reflect.KProperty
+
+// Single get
+public operator fun GeneralCommandMessage.get(name: String): String? =
+	arguments.entries.firstOrNull { it.key.contentEquals(name, true) }?.value
```

Comment:
Should maybe multi-line this, make the `it` specific, and prepend a `ignoreCase=` before the `true`.